dist/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+<body>
+    <br>
+    <div class="container">
+    <div><h3> Today's To Do</h3>
+        <form action="">
+            <div class="input-icons">
+            <button id="newBook" class="btn"><i class="fa fa-save"></i></button>
```

Comment:
- [ ] please set attribute to your button

Reply:
Ok, Thanks @henrykgggg  noted and changed accordingly